tests/test_tokenization_albert.py

```diff
@@ -102,3 +102,48 @@ def test_sequence_builders(self):
         assert encoded_pair == [tokenizer.cls_token_id] + text + [tokenizer.sep_token_id] + text_2 + [
             tokenizer.sep_token_id
         ]
+
+    @slow
+    def test_tokenizer_integration(self):
+        tokenizer_classes = [self.tokenizer_class]
+        if self.test_rust_tokenizer:
+            tokenizer_classes.append(self.rust_tokenizer_class)
+
+        for tokenizer_class in tokenizer_classes:
+            tokenizer = tokenizer_class.from_pretrained("albert-base-v2")
+
+            sequences = [
+                "ALBERT: A Lite BERT for Self-supervised Learning of Language Representations",
+                "ALBERT incorporates two parameter reduction techniques",
+                "The first one is a factorized embedding parameterization. By decomposing the large vocabulary embedding matrix into two small matrices, we separate the size of the hidden layers from the size of vocabulary embedding."
+            ]
+
+            encoding = tokenizer(sequences, padding=True)
```

Comment:
(nit & question) what is the recommend way of handling `padding`? Isn't it rather `padding="longest"`?

Reply:
`True` is `longest`, `False` is `do_not_pad`, and then there's `max_length`. Either of those 5 work, and they're all mentioned in the docs so I would argue they're all recommended.